mlflow/store/artifact/cloud_artifact_repo.py

```diff
@@ -29,6 +29,15 @@
 _ARTIFACT_UPLOAD_BATCH_SIZE = (
     50  # Max number of artifacts for which to fetch write credentials at once.
 )
+_AWS_MIN_CHUNK_SIZE = 5 * 1024**2  # 5 MB is the minimum chunk size for S3 multipart uploads
+
+
+def _validate_chunk_size_aws() -> None:
+    chunk_size = MLFLOW_MULTIPART_UPLOAD_CHUNK_SIZE.get()
```

Comment:
```suggestion
def _validate_chunk_size_aws(chunk_size: int) -> None:
```

Can we pass chunk size to this function so the code would look like this?


```python
chunk_size = ...
_validate_chunk_size_aws(chunk_size)
```

Reply:
```suggestion
def _validate_chunk_size_aws() -> int:
```

We can return the passed-in chunk if that's convenient.

Reply:
Changed to take the chunk size as a parameter. Kept None output for simpler testing / mocking setups.